assets/elasticsearch/Dockerfile

```
@@ -0,0 +1,24 @@
+
+RUN apt-get update \
+    && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+    elasticsearch=8.6.2 \
+    libcurl4 \
+    libcurl4-openssl-dev \
+    python3-pip \
+    python3-venv \
+    tmux \
+    vim;
+
+RUN pip3 install elasticsearch==8.6.2 requests;
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)